charactersheet/charactersheet/viewmodels/root.js

```diff
@@ -127,6 +127,49 @@ function RootViewModel() {
 		self.activeTab('players');
 	};
 
+	//Tab shortcut Methods
```

Comment:
Good catch. There was this weird merge when I updated the branch. I guess that was an artifact.